locales/locales.php

```diff
@@ -1998,6 +1998,18 @@ public function __construct() {
 		$pcd->nplurals = 2;
 		$pcd->plural_expression = 'n > 1';
 
+		$pa_pk = new GP_Locale();
+		$pa_pk->english_name = 'Punjabi (Pakistan)';
+		$pa_pk->native_name = 'پنجابی';
+		$pa_pk->lang_code_iso_639_1 = 'pa';
+		$pa_pk->lang_code_iso_639_2 = 'pnb';
```

Comment:
```suggestion
		$pa_pk->lang_code_iso_639_2 = 'pan';
		$pa_pk->lang_code_iso_639_3 = 'pnb';
```

`pnb` is the ISO 3 code, see https://iso639-3.sil.org/code/pnb.

Reply:
You're right, thanks!